pyfr/libs/ascent.py

```diff
@@ -0,0 +1,45 @@
+# -*- coding: utf-8 -*-
+
+from ctypes import c_void_p, RTLD_GLOBAL
+
+from pyfr.ctypesutil import LibWrapper
+from pyfr.libs.conduit import ConduitNode
+
+class AscentWrappers(LibWrapper):
+    _libname = 'ascent'
+
+    # Functions
+    _functions = [
+        (c_void_p, 'ascent_close', c_void_p),
+        (c_void_p, 'ascent_create', c_void_p),
+        (c_void_p, 'ascent_execute', c_void_p, c_void_p),
+        (c_void_p, 'ascent_open', c_void_p, c_void_p),
+        (c_void_p, 'ascent_publish', c_void_p, c_void_p),
+    ]
+
+    def __init__(self):
+        super().__init__(mode=RTLD_GLOBAL)
+
+class Ascent:
+    def __init__(self):
+        self.lib = AscentWrappers()
+        self.ptr = c_void_p(self.lib.ascent_create(c_void_p()))
+
+    def __del__(self):
```

Comment:
Check ascent is actually loaded as `__del__` will be called even if we could not wrap Ascent.